Casks/araxis-merge.rb

```diff
@@ -9,10 +9,14 @@
     url "https://www.araxis.com/download/Merge#{version}-macOS.dmg"
   end
 
-  appcast "https://www.araxis.com/news-feed.atom"
   name "Araxis Merge"
   homepage "https://www.araxis.com/merge/"
 
+  livecheck do
+    url "https://www.araxis.com/news-feed.atom"
+    regex(/Araxis Merge\s+(\d+\.\d+)\s+for macOS/i)
```

Comment:
```suggestion
    url "https://www.araxis.com/merge/download.en"
    regex(/href=.*?Merge[._-]?v?(\d+(?:\.\d+)+)-macOS\.dmg/i)
```

The Merge download page links to the disk image file, so it would arguably be better to check that.

The standard regex for versions like `1.2`/`1.2.3`/etc. works for `2020.5479`, so it wouldn't hurt to use it here. Standardizing on common regex components means that we can easily substitute them with a built-in regex snippet if/when we implement this in the future.